src/WellIO.jl

```diff
@@ -1,5 +1,20 @@
 module WellIO
 
-# Write your package code here.
+using FileIO
+using Tables
 
+function load(filepath::String)
+    file = FileIO.query(filepath)
+    data = open(file) do f
+           read(f, String)
+        end
+    version_block = match(r"~V(.*?)~"s, data)
+    wellinfo_block = match(r"~W(.*?)~"s, data)
+    curveinfo_block = match(r"~C(.*?)~"s, data)
+    otherinfo_block = match(r"~O(.*?)~"s, data)
+    datatable_block = match(r"~A(.*?)\Z"s, data)
```

Comment:
What I don't like about this approach is that you are parsing the full file multiple times to get tiny sections of the header. Can you please refactor the code so that you read the entire header once in a variable, and then do the regex only with the header instead of the entire file? The tables that come after can be really big so depending on the regex things can be unnecessarily slow.